deploy/cluster_role.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
nit/note for follow up: we should rename this to globalnet_cluster_role so the filename it's clearer

Reply:
This was also discussed with Stephen. Currently we're using the same SA as submariner engine and routeagent. So kept the name generic in case need to add more permissions to cluster roles. Common name also conveys message that this cluster role applies to engine and route-agent too, as using same SA. Using globalnet_cluster_role without a different SA can give misleading impression that this applies only to globalnet.

IMO, this should be renamed/refactored when we move to using individual and specific SAs for engine, routeagent and globalnet.

Reply:
Makes sense, thanks for the explanation, agreed